src/docs/patterns/tables.mdx

```diff
@@ -0,0 +1,463 @@
+        `DONT: Do not surprise users by hiding high priority actions in the expanded row space.`,
+    ]}
+/>
+
+## Horizontal Scrolling
+
+When your table overflows horizontally, you should allow your users to scroll horizontally to view more about the table. However, this interaction is less ergonomically convenient when users are not using a touch screen or a trackpad. See the [Responsive](#responsive) section below.
+
+The application may display a scroll bar to suggest that there are more contents to the right.
+
+<ImageGrid
+    images={[HorizontalScrollBar]}
+    caption={[`Place horizontal scroll bar between rows and the footer / pagination area.`]}
+    regularWidth
+/>
+
+<ImageGrid
+    images={[StickyCheckbox, StickyRowAction]}
+    captionsUnderImages={[
+        `The checkbox column should be sticky to the left as the user scrolls to the right.`,
+        `The action column, if designed to be always visible, should be sticky to the right. Do not include too many row actions that cover excessive table body area.`,
+    ]}
+/>
+
+## Searching and Filtering
+
+There are many different ways to perform searching and filtering with table data. Below are a few examples:
+
+<ImageGrid
+    images={[Filter1, Filter2, Filter3, Filter4]}
+    captionsUnderImages={[
+        `A simple search field embedded in a table.`,
+        `A search field outside a table.`,
+        `Filter criteria listed as chips.`,
+        `A "saved filter" dropdown.`,
+    ]}
+/>
+
+<ImageGrid
+    images={[AdvancedTableFilter]}
+    caption={[
+        `You may combine multiple interactions (searching and filtering) and input methods (keyboard navigations, keyboard typing, mouse clicking) together for an "advanced search".`,
+    ]}
+    regularWidth
+/>
+
+## Transition Animation
+
+Often, table data look highly similar between rows and can disorient your users. You are encouraged to use animations in your applications to help users understand certain transitions.
+
+<ImageGrid
+    images={[AdditionAnimation, SubstractionAnimation]}
+    captionsUnderImages={[
+        `Transition animation for adding a row to the table. Consider using a background color to temporarily highlight the newly-added row.`,
+        `Transition animation for removing a row from the table.`,
+    ]}
+/>
+
+<Divider />
+
+# Tables vs. Lists
+
+<ImageGrid
+    images={[ListVTableTable, ListVTableList]}
+    caption={['A same set of data may be presented in tables (#1) or lists (#2), depends on the use cases.']}
+/>
+
+In most cases, we encourage you to organize your data into lists instead of tables. However, there is no clearly defined boundary between lists and tables. Below are a few factors you should consider when deciding how you want to present your data.
+
+## Arrangement
+
+Tables expose each data point into their own columns, allowing users to easily look up a row starting from any column. Lists, on the other hand, group their content row-by-row.
```

Comment:
```suggestion
Tables place more emphasis on columns, allowing users to easily find a row by any of its associated data points. Lists, on the other hand, can be less structured and tend to treat each item as a cohesive whole rather than a collection of individual data values.
```